runtimes/common/src/migrations.rs

```diff
@@ -58,12 +71,39 @@ where
 	}
 
 	#[cfg(feature = "try-runtime")]
-	fn post_upgrade(_state: sp_std::vec::Vec<u8>) -> Result<(), &'static str> {
+	fn post_upgrade(_state: sp_std::vec::Vec<u8>) -> Result<(), TryRuntimeError> {
+		use frame_support::ensure;
+
 		log::info!("RemoveInsecureRandomnessPallet::post_upgrade() checks 🔍");
-		if frame_support::migration::have_storage_value(PALLET_RUNTIME_NAME, PALLET_STORAGE_NAME, b"") {
-			Err("Storage in pallet_insecure_randomness_collective_flip is not empty after migration.")
-		} else {
-			Ok(())
-		}
+
+		ensure!(
+			!frame_support::migration::have_storage_value(PALLET_RUNTIME_NAME, PALLET_STORAGE_NAME, b""),
+			TryRuntimeError::Other(
+				"Storage in pallet_insecure_randomness_collective_flip is not empty after migration.",
+			)
+		);
+		Ok(())
+	}
+}
+
+pub struct BumpStorageVersion<T>(PhantomData<T>);
```

Comment:
Can you add a short explanation plus relevant link on why this is required?

Reply:
Add a comment here: [9c36c92](https://github.com/KILTprotocol/kilt-node/pull/541/commits/9c36c920531974ae4a5beb1ebca82d31944052d0) 

Hope it is clear what I want to say